person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services
+    is_of_age? || @parent_permission
+  end
```

Comment:
- [ ] The second change requested by the previous reviewer only demonstrated how to use the `private` using the present state of your code. And you implemented that well enough. However, you still need to implement his first change which is using the correct method name, which is `of_age?`, not `is_of_age?`. As you can see I get errors when I call the method

  ![pers-1](https://user-images.githubusercontent.com/36542986/186131792-f45c151a-bdec-4adf-88b7-eacde934a227.jpg)